cves/kernel/CVE-2018-12904.yml

```diff
@@ -343,8 +219,56 @@ stacktrace:
     what your answer was.
   any_stacktraces:
   stacktrace_with_fix:
+description: >-
+  When virtualizing a system, a processor is effectively "emulated". For
+  example, a 64-bit processor can emulate a 32-bit one. This can include giving
+  oneself more privilege that normally allowed, because the processor is
+  "sandboxed".
+
+
+  In this case, an intel processor can emulate another. If you crash the virtual machine in a particular way, some of your privileges remained on the main system and aren't properly revoked. This can be used to escalate privileges on the host system and do more nefarious things from there.
+unit_tested:
+  fix:
+  code:
+  question: |
+    Were automated unit tests involved in this vulnerability?
+    Was the original code unit tested, or not unit tested? Did the fix involve
+    improving the automated tests?
+
+    For code: and fix: - your answer should be boolean.
+
+    For the code_answer below, look not only at the fix but the surrounding
+    code near the fix in related directories and determine if and was there were
+    unit tests involved for this subsystem.
+
+    For the fix_answer below, check if the fix for the vulnerability involves
+    adding or improving an automated test to ensure this doesn't happen again.
+  fix_answer:
+  code_answer:
+reported_date:
+specification:
   note:
+  answer:
+  instructions: |
+    Is there mention of a violation of a specification? For example, the POSIX
+    spec, an RFC spec, a network protocol spec, or some other requirements
+    specification.
+
+    Be sure to check the following artifacts for this:
+      * bug reports
+      * security advisories
+      * commit message
+      * mailing lists
+      * anything else
+
+    The answer field should be boolean. In answer_note, please explain
+    why you come to that conclusion.
+announced_date: 2018-06-27
+curation_level: 0
```

Comment:
Your curation level was supposed to be set to 2. This is why your CI doesn't fail when most of your stuff isn't even filled out